shared/actions/login/saga.js

```diff
@@ -488,7 +493,10 @@ function* initalizeMyCodeStateForAddingADevice() {
 }
 
 function* startLoginSaga() {
-  yield put(navigateAppend(['usernameOrEmail'], [loginTab, 'login']))
+  yield put(Creators.setLoginFromRevokedDevice(''))
```

Comment:
the component used to dispatch all these actions but really they should be one thing. i moved this here but really we should likely just listen to the startlogin action in the reducer and clear all this stuff out